src/content.js

```diff
@@ -32,6 +32,7 @@ window.$ = $;
 window.select = select;
 
 const repoUrl = pageDetect.getRepoURL();
+const isMac = /Mac/i.test(navigator.platform);
```

Comment:
This should go in https://github.com/sindresorhus/refined-github/blob/master/src/libs/utils.js as a reusable method.

Reply:
Okay. As a method or just an attribute, since the value never changes?